nipype/interfaces/niftyseg/maths.py

```diff
@@ -0,0 +1,427 @@
+                              xor=['operand_value', 'operand_file'],
+                              desc=desc)
+
+
+class BinaryMaths(MathsCommand):
+    """Interface for executable seg_maths from NiftySeg platform.
+
+    Interface to use any binary mathematical operations that can be performed
+    with the seg_maths command-line program. See below for those operations:
+        -mul    <float/file>    Multiply image <float> value or by other image.
+        -div    <float/file>    Divide image by <float> or by other image.
+        -add    <float/file>    Add image by <float> or by other image.
+        -sub    <float/file>    Subtract image by <float> or by other image.
+        -pow    <float>         Image to the power of <float>.
+        -thr    <float>         Threshold the image below <float>.
+        -uthr   <float>         Threshold image above <float>.
+        -smo    <float>         Gaussian smoothing by std <float> (in voxels
+                                and up to 4-D).
+        -edge   <float>         Calculate the edges of the image using a
+                                threshold <float>.
+        -sobel3 <float>         Calculate the edges of all timepoints using a
+                                Sobel filter with a 3x3x3 kernel and applying
+                                <float> gaussian smoothing.
+        -sobel5 <float>         Calculate the edges of all timepoints using a
+                                Sobel filter with a 5x5x5 kernel and applying
+                                <float> gaussian smoothing.
+        -min    <file>          Get the min per voxel between <current> and
+                                <file>.
+        -smol   <float>         Gaussian smoothing of a 3D label image.
+        -geo    <float/file>    Geodesic distance according to the speed
+                                function <float/file>
+        -llsnorm  <file_norm>   Linear LS normalisation between current and
+                                <file_norm>
+        -masknan <file_norm>    Assign everything outside the mask (mask==0)
+                                with NaNs
+        -hdr_copy <file>        Copy header from working image to <file> and
+                                save in <output>.
+        -splitinter <x/y/z>     Split interleaved slices in direction <x/y/z>
+                                into separate time points
+
+    For source code, see http://cmictig.cs.ucl.ac.uk/wiki/index.php/NiftySeg
+    For Documentation, see:
+        http://cmictig.cs.ucl.ac.uk/wiki/index.php/NiftySeg_documentation
+
+    Examples
+    --------
+    >>> from nipype.interfaces import niftyseg
+    >>> node = niftyseg.BinaryMaths()
+    >>> node.inputs.in_file = 'im1.nii'
+    >>> node.inputs.operation = 'sub'
+    >>> node.inputs.operand_file = 'im2.nii'
+    >>> node.inputs.output_datatype = 'float'
+    >>> node.cmdline  # doctest: +ALLOW_UNICODE
+    'seg_maths im1.nii -sub im2.nii -odt float im1_sub.nii'
```

Comment:
I think all these interfaces would deserve some more doctests with more operations, so that we are sure that they work and that we don't break them in future maintenance.